src/Diagnostics.ModelsAndUtils/Attributes/ResourceFilter/AppServiceCertificateFilter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Attributes
+{
+    /// <summary>
+    /// Class representing Filter for App service certificate
+    /// </summary>
+    public class AppServiceCertificateFilter : ResourceFilterBase
+    {
+        public AppServiceCertificateFilter(bool internalOnly = true) : base(ResourceType.AppServiceCertificate, internalOnly)
```

Comment:
Are these constructors necessary?

Reply:
Ya, it is. The base constructor has the Resource Type

Reply:
True, not sure you need internalOnly as a parameter